ops/docker/Dockerfile.packages

```diff
@@ -107,7 +107,7 @@ ENTRYPOINT ["npm", "run", "start"]
 
 FROM base as drippie-mon
 WORKDIR /opt/optimism/packages/drippie-mon
-ENTRYPOINT ["npm", "run", "start"]
+ENTRYPOINT ["npm", "run", "start:drippie-mon"]
```

Comment:
looks wrong

Reply:
I changed the start script so you can selectively run services: https://github.com/ethereum-optimism/optimism/pull/4804/files#diff-1f6ff7ce7dfe34ffbf9dfdc05039ecf20a9d5b677d4e902037dbc88ac6d11bccR12